submissions/korotushak96/js-dom/script.js

```diff
@@ -0,0 +1,90 @@
+            description: 'A Niffler was a creature with a long snout and a coat of black, fluffy fur. They were attracted to shiny things, which made them wonderful for locating treasure, but this also means that they could wreak havoc if kept (or set loose) indoors. Nifflers in general were usually harmless.' ,
+            image: 'img/niffler.png'
+        },
+        {
+            id: 3,
+            name: 'Bowtruckle',
+            description: 'The Bowtruckle is a hand-sized, insect-eating tree dweller with long sharp fingers (two on each hand), brown eyes, and a general appearance of a flat-faced stick figure made of bark and twigs, which serves well as camouflage in its natural habitat. They are immensely difficult to spot.',
+            image: 'img/Bowtruckle.png'
+        },
+        {
+            id: 4,
+            name: 'Nundu',
+            description: 'The Nundu is a large East African beast that resembles a leopard. It moves silently, despite its "gigantic" size and is considered by some to be the most dangerous creature alive. The breath of the Nundu is toxic and filled with disease so potent that it can wipe out entire villages of people',
+            image: 'img/nundu_by_bear_drool_dbd2elu-pre.png'
+        },
+        {
+            id: 5,
+            name: 'Jarvey',
+            description: 'The Jarvey resembles an overgrown ferret, and is commonly found in Great Britain, Ireland, and North America. The Jarvey is capable of Human speech, although true conversation with a Jarvey is impossible. The creature uses short, usually rude, statements and phrases in an almost constant stream.',
+            image: 'img/Jarvey.png'
+        },
+        {
+            id: 6,
+            name: 'Murtlap',
+            description: 'The Murtlap is a marine beast resembling a rat with a growth on its back resembling a sea anemone, found on the coastal areas of Britain.',
+            image: 'img/Murtlap.jpg'
+        },
+    ]
+
+    let menuHTML = '';
+    let descriptionHTML = '';
+    const beastsDescriptionsFrag = document.createDocumentFragment();
+
+    beastsData.forEach((beast)=>{
+        const {id, name, description, image} = beast;
+        // create menu-items
+        menuHTML += `<div class='menu-item'>${name}</div>`;
+        // and description
+        descriptionHTML += 
+            `<div class='content-item'>
+                <div class='content-item_text'>${description}</div>
+                <img class='content-item_image' src='${image}'></img>
+            </div>`
+    });
+
+    menu.insertAdjacentHTML('afterbegin', menuHTML);
+    content.insertAdjacentHTML('afterbegin', descriptionHTML);
+
+    const contentItem = document.querySelectorAll('.content-item');
+    const menuItem = document.querySelectorAll('.menu-item');
+
+    burger.addEventListener('click', ()=>{
+        burger.classList.toggle('change');
+        menu.classList.toggle('open');
+    });
+
+    switchActive=(i)=>{
```

Comment:
Here you use two `forEach` loops. It's also very expensive operations for performance.
What do you think about using just one loop and also to use `data-*` attributes in your menu links?

Reply:
Please, explaine why do we need data-* property?

Reply:
For example, you could check data attribute from clicked link and make a comparison with beasts `id`.

Reply:
I think we may do without it, may not we?

Reply:
Ok, it's up to you. Maybe you are tired after all previous fixes, but here we trying to improve our skills in web development and to write code cleaner and better.
At this moment you have already did a great job! And your decision is really good and it works properly at a glance.
But I believe you could make your code better if you replace two `forEach` to `find` method. See more [here](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/find)
Thank you for understanding!
Please do this fix and we are ready for merging this pull request!

Reply:
Thanks for your help. I don't even think about find method. I"ll  implement it